fulp_modules/features/toys/code/plushies.dm

```diff
@@ -55,25 +55,25 @@
 	attack_verb_simple = list("jumpscare")
 	squeak_override = list('fulp_modules/features/toys/sound/jumpscare.ogg'= 1)
 
-/obj/item/toy/plush/chica
+/obj/item/toy/plush/freddy/chica
```

Comment:
/freddy/chica

can u change it so its like animatronic instead of freddy or smth